### ISSUE TITLE
While getting the multiple classes per level up thing working could you make this work as before...?

### ISSUE THREAD

**Issue by Panxodakilla** [cabarius/ToyBox#67]
Could you make it like it was in beta, so that when lowering level on party editor to get more level ups kept the overall character level after loads? Now, you can do this to get to take multiple level ups as before, but the character level resets to "pre-edited" after loading a save and you have to manually click reset

**Comment by dimhaku** [comment]
I also agree with this. This also has more flexibility than a selected multiple classes per level up since it allows people to pick and choose different classes at each level up.

**Comment by quasihtonder** [comment]
Agree on this too.
